.github/workflows/release.yml

```diff
@@ -35,6 +31,45 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
         run: npx semantic-release
+  update-core-repo:
+    needs: release-npm-package
+    name: Update package in core repo
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out core repo
+        uses: actions/checkout@v2
+        with:
+          repository: ministryofjustice/bichard7-next-core
+          fetch-depth: 0
+          token: ${{ secrets.GH_ACCESS_TOKEN }}
+      - name: Update next data package
+        run: npm install @moj-bichard7-developers/bichard7-next-data@latest
+      - name: Create Pull Request
+        uses: peter-evans/create-pull-request@v4
+        with:
+            token: ${{ secrets.GH_ACCESS_TOKEN }}
+            commit-message: Update bichard7-next-data NPM package
+            title: Update bichard7-next-data NPM package
+            body: |
+              - Update bichard7-next-data NPM package
+  
+              Auto-generated by GitHub Actions
+            branch: update-bichard7-next-data-package
+  release-maven-package:
+    name: Release Maven package
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Install Java and Maven
+        uses: actions/setup-java@v1
+        with:
+          java-version: 17
+          server-id: ossrh
+          server-username: MAVEN_USERNAME
+          server-password: MAVEN_PASSWORD
```

Comment:
Are these things that should be hidden away as github secrets?

Reply:
Yes they are as far as I can see: https://github.com/ministryofjustice/bichard7-next-data/blob/f253bd0b0af4d12ee1652dfbab4cf6c096178d55/.github/workflows/release.yml#L80